src/zcl_abapgit_transport_mass.clas.abap

```diff
@@ -0,0 +1,70 @@
+CLASS zcl_abapgit_transport_mass DEFINITION
+  PUBLIC
+  INHERITING FROM zcl_abapgit_transport
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    CLASS-METHODS run .
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_TRANSPORT_MASS IMPLEMENTATION.
+
+
+  METHOD run.
+
+    DATA:
+      lt_trkorr          TYPE trwbo_request_headers.
+
+    DATA:
+      lo_transport_zipper TYPE REF TO lcl_transport_zipper,
+      lo_except           TYPE REF TO cx_root,
+      lv_folder           TYPE string.
+
+    TRY.
+
+        lcl_gui=>select_tr_requests( IMPORTING et_trkorr = lt_trkorr ).
+
+        IF lt_trkorr[] IS NOT INITIAL.
+
+* Call destination folder popup
+          lcl_gui=>f4_folder( CHANGING cv_folder = lv_folder ).
```

Comment:
RETURNING?

Reply:
Yes. This method is not used in a dedicated selection screen anymore so it can be turned to a RETURNING parameter. The changing was a remaining of its previous use case ( because if the user cancel the popup, we usually don't want the previous value to be erased ).